descwl_shear_sims/simple_sim.py

```diff
@@ -101,13 +101,27 @@ class Sim(object):
     se_dim: int
         Dimensions of the single epoch image.  If None (the default) the
         size is chosen to encompass the coadd for small dithers.
+    layout_type : str, optional
+        A string indicating how to layout the objects in the image. Possible options
+        are
+
+            'random' : randomly place the objects in the image
+            'grid' : place the objects on a grid in the image
+
+        Default is 'random'.
+    layout_kws : dict, optional
+        A dictionary giving options for the specific layout type. These vary per
+        layout type and are
+
+            'random' - no options
+            'grid' - any of the following
+                'dim' : int, required
+                    An integer giving the dimension of the grid on one axis.
+
+    grid_objects : int, optional
```

Comment:
grid_objects sounds like a boolean but it is an integer

Reply:
This one shouldn't even be here. Will remove.